api_endpoints/README.md

```diff
@@ -0,0 +1 @@
+# api_endpoints
```

Comment:
Could you please add a step by step instructions on how your endpoints can be tested.